tests/test_api.py

```diff
@@ -259,6 +269,7 @@ def test_unknown_field(self, client, entities, endpoint, method):
         ('/participants', 'phenotypes'),
         ('/studies', 'study_files'),
         ('/families', 'participants'),
+        ('/sequencing-centers', 'biospecimens')
     ])
     def test_relations(self, client, entities, resource, field):
```

Comment:
Looks like you're also missing a couple tests for sequencing centers in `test_bad_input` and `test_missing_required_params`